android/src/main/java/com/tailscale/ipn/ui/view/AboutView.kt

```diff
@@ -30,8 +30,9 @@ import com.tailscale.ipn.R
 import com.tailscale.ipn.ui.Links
 
 @Composable
-fun AboutView(nav: BackNavigation) {
-  Scaffold(topBar = { Header(R.string.about_view_title, onBack = nav.onBack) }) { innerPadding ->
+fun AboutView(backToSettings: BackNavigation) {
```

Comment:
nit:  There's an assumption in the naming here.  The caller sets up the back nav stack.  It does not have to be "Settings".

Reply:
This was intentional - the caller is actually explicitly making this navigate back to settings, not just generally "back".